### ISSUE TITLE
cannot share sheet and transfer ownership

### ISSUE THREAD

**Issue by skikut** [Squarespace/datasheets#18]
`returned "The transferOwnership parameter must be enabled when the permission role is 'owner'.">`

I would like to transfer ownership using the workbook.share method: `workbook.share(email=email, role='owner', notify=True)`

**Comment by sl33t** [comment]
@zcmarine Could this be solved by adding the transferOwnership parameter to the new_permission variable if the role is ownership?

**Comment by zcmarine** [comment]
@sl33t Yeah, it looks like it may be as simple as that. I believe we just add it as a parameter to the .create() method [here](https://github.com/Squarespace/datasheets/blob/master/datasheets/workbook.py#L101) based on the [Drive API docs](https://developers.google.com/drive/api/v3/reference/permissions/create). Let me know if either of you want to take a shot at that and verify it works as expected.

**Comment by skikut** [comment]
@zcmarine i have the fix on my local, let me know if you would like me to make the pull request. it was an easy fix
